src/components/lobby/game_lobby.tsx

```diff
@@ -88,14 +91,32 @@ const GameLobby = ({ ws, game, gameRef, setGame }: GameLobbyProps): JSX.Element
     setGame(newGame);
   };
 
+  /**
+   * For LEAVE_GAME
+   * @param payload LeaveGameJSON
+   */
+  const leaveGame = (payload: unknown): void => {
+    const data = payload as LeaveGameJSON;
+    const { success, game: g, error } = data;
```

Comment:
is `g` an alias? never use this before, so I'm just curious

Reply:
you can use aliases if there are variable name conflicts